exchange.js

```diff
@@ -5,7 +5,14 @@ let GBPrate
 const getExchangeRates = async () => {
     return fetch('https://www.dolarsi.com/api/api.php?type=cotizador')
         .then(r => r.json())
-        .then(data => data)
+        .then(data => {
+            // Saving the last time the data was fetched since the API does not return any dates
+            const timestamp = new Date()
+            const timestampMessage = `Fecha de cotizacion: ${timestamp.getDate()} / ${timestamp.getMonth()} / ${timestamp.getFullYear()}  ${timestamp.getHours()} : ${timestamp.getMinutes()}`
+            localStorage.setItem('timestampMessage', JSON.stringify(timestampMessage))
```

Comment:
Since timestampMessage is already a string, there's no need to stringify it. You can just store it as it is

Reply:
done